### ISSUE TITLE
bump version

### ISSUE THREAD

**Issue by miniplus** [mledoze/countries#263]
Last version (1.8.1) dates back from Oct 24, 2016. There's been a few commits since so perhaps it's time to bump the version?

**Comment by mledoze** [comment]
@miniplus you're right, it's time for the version 1.9. I'm on it.

**Comment by mledoze** [comment]
The new release v1.9.0 is almost ready. I'm waiting for a few PRs to be included in the release.

**Comment by blumk** [comment]
@mledoze I'd release it as v2.0.0 since it contains breaking changes (e.g. PR #266 "multiple capitals")
[SemVer](https://semver.org/):

> MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.

**Comment by mledoze** [comment]
@blumk yes you're right but I want also want to drop the support for PHP 5 in the upcoming release v2.0.0, so that's why I want to release it as v1.9.0 first.

**Comment by mledoze** [comment]
> I'd release it as v2.0.0 since it contains breaking changes

@blumk I gave it a second thought and I will finally release it as v2.0.0 because I also want to gradually drop the support for PHP 5. So here is the roadmap for the next 2 major releases:

- v2.0.0 will include all the changes from the master branch and will require the latest version of PHP 5 which currently is 5.6.33.
- v3.0.0 will require the latest version of PHP 7, which is 7.2. I also plan to add unit tests (with 100% code coverage) in this release.
